core/prisma/migrations/20250313193528_add_see_extra_pub_values_capability/migration.sql

```diff
@@ -0,0 +1,37 @@
+-- AlterEnum
+ALTER TYPE "Capabilities" ADD VALUE 'seeExtraPubValues';
+COMMIT;
+
+INSERT INTO
+    "membership_capabilities"
+VALUES
+    (
+        'editor'::"MemberRole",
+        'pub'::"MembershipType",
+        'seeExtraPubValues'::"Capabilities"
+    ),
+    (
+        'admin'::"MemberRole", 
+        'pub'::"MembershipType", 
+        'seeExtraPubValues'::"Capabilities"
+    ),
+    (
+        'editor'::"MemberRole",
+        'stage'::"MembershipType",
+        'seeExtraPubValues'::"Capabilities"
+    ),
+    (
+        'admin'::"MemberRole", 
+        'stage'::"MembershipType", 
+        'seeExtraPubValues'::"Capabilities"
+    ),
+    (
+        'editor'::"MemberRole",
+        'community'::"MembershipType",
+        'seeExtraPubValues'::"Capabilities"
+    ),
+    (
+        'admin'::"MemberRole", 
+        'community'::"MembershipType", 
+        'seeExtraPubValues'::"Capabilities"
+    );
```

Comment:
related note: i think it would be nice if we could have some way we could declare these capabilities in code and have them auto added to a migration if we change them. i already found it hard to keep track of them when they were in one migration file, now that they are starting to split into many it's going to be even harder! ill spend some time thinking about this (would also really like a solution for migrating stored procedures and triggers)